src/MoBi.Presentation/MenusAndBars/ContextMenus/ContextMenuForHistoricalResultsNode.cs

```diff
@@ -59,8 +59,12 @@ public IContextMenu InitializeWith(DataRepository dataRepository, IMoBiSimulatio
             .WithIcon(ApplicationIcons.Rename)
             .WithCommand(command));
 
+         var removeResultsCommand = _container.Resolve<RemoveSimulationResultCommand>();
+         removeResultsCommand.Subject = dataRepository;
+         removeResultsCommand.Simulation = simulation;
```

Comment:
See on line 49 the same pattern except initializing the properties is done with a method `InitializeWith`. It's exactly what you have, but in a method.